chromadb/proto/convert.py

```diff
@@ -19,42 +19,43 @@
     VectorEmbeddingRecord,
     VectorQueryResult,
 )
+import numpy as np
+from numpy.typing import NDArray
```

Comment:
Could you elaborate on how it might be unsafe? Not sure I understand. And you're suggesting running staging tests and not writing a new suite to test this, correct?

Reply:
I mean `NDArray`, and I mean testing that `NDArray` works across numpy versions because I remember this explodes sometimes.

Reply:
I can look into implementing a framework like `tox` to run our tests across package versions. Before I do so, however, do you remember how long ago you saw version issues with `NDArray` and what they looked like? We require `numpy>=1.22.5 ` and `NDArray` is available as of `1.21`. I'm wondering if issues you recall were because we previously allowed `numpy` versions earlier than that.

Reply:
When did we bump our numpy version? I just want to be sure.

Reply:
It appears that we first started explicitly requiring it here: https://github.com/chroma-core/chroma/commit/a3d25c821c69ad7bf7985d146db52836b8059e5f. Before that, I assume it would've been installed through `pandas`, and `pandas 1.3.5` required `numpy 1.17.3`